dspy/__init__.py

```diff
@@ -26,4 +26,55 @@
 HFModel = dsp.HFModel
 
 configure = settings.configure
-context = settings.context
\ No newline at end of file
+context = settings.context
+
+
+import json
+import os
+import openai
+
+class Models:
```

Comment:
This is quite creative and convenient but it probably won't work in the grand scheme of things, at least not without a lot of effort to expand then maintain.

Basically there are at least 10 providers people use, not all of them in DSPy yet but many are.

Reply:
create_lm could call any model provider you want. Mainly, I wanted to solve the problem of LM configuration saving and loading. You can't serialize the LM so you need to save the kwargs and easily map it to an LM. Any other idea on how to solve this?